internal/cli/clusters/clusters.go

```diff
@@ -138,6 +138,36 @@ func removeReadOnlyAttributes(out *atlasClustersPinned.AdvancedClusterDescriptio
 	}
 }
 
+func removeReadOnlyAttributesLatest(out *atlasv2.ClusterDescription20240805) {
```

Comment:
What does latest mean in this context? SDK? could you add a comment to clarify?

Reply:
yes, we are using latest everywhere to indicate the latest cluster definition, I can add! thx